desktop/main/walletFile.ts

```diff
@@ -153,7 +153,7 @@ export const encryptWallet = async (
 
 export const loadRawWallet = async (path: string): Promise<WalletFile> => {
   const fileContent = await fs.readFile(path, { encoding: 'utf8' });
-  return JSON.parse(fileContent) as WalletFile;
+  return JSON.parse(fileContent);
```

Comment:
After implementing guards, do:
```suggestion
  const content = JSON.parse(fileContent);
  if (!isWalletFile(content)) {
    throw new Error(`File ${path} is not a valid wallet file`);
  }
  return content;
```

No more changes are needed.